packages/cadl-ranch-specs/http/routes/main.tsp

```diff
@@ -0,0 +1,476 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+/**
+ * Define scenario in building the http route/uri
+ */
+@scenarioService("/routes")
+namespace Routes;
```

Comment:
open to a different name here too

Reply:
there's also overlap here with the parameters grouping of tests: I think it would be better to add these uri tests to existing `query`, `path` etc sections, what are people's thoughts

Reply:
While technically I could write a full template with path and query at the same time which would make a generic "route" better, I feel you're right that we should probably split by path/query. I feel like the full template will be rare if happening at all, so I'm ok to keep it as a special case of "path"

Reply:
well I think it is good to have all of uri template here, uri template kinda abstract away the fact that you have query and path parameter at this point, you just have params to build the uri.

I think it is nicer to have an overall view of what uri template is supported by emitters than having to fetch in every single category. I am however down to cleanup the other ones as support for uri template is added

Reply:
that's fair, thanks @timotheeguerin